aptos-move/framework/supra-framework/sources/supra_governance.spec.move

```diff
@@ -57,21 +57,22 @@ spec supra_framework::supra_governance {
         min_voting_threshold: u128,
         required_proposer_stake: u64,
         voting_duration_secs: u64,
+        voters: vector<address>,
```

Comment:
If this is copied from orignal `spec` file, delete this. The spec won't be consistent with the implementation.

Reply:
Okay, I have deleted it